app/components/affiliate-page/create-payout-modal.hbs

```diff
@@ -3,6 +3,13 @@
     Initiate payout
   </div>
 
+  <div class="prose mb-8">
+    <span class="font-bold">Note</span>: PayPal might charge you a fee for receiving payments. This can be as high as 10% depending on the country
```

Comment:
```suggestion
    <b>Note</b>: PayPal might charge you a fee for receiving payments. This can be as high as 10% depending on the country
```

(Within `prose`, try to use standard HTML tags so that the default styling just works)